src/main/java/organice/logic/commands/DoneCommand.java

```diff
@@ -0,0 +1,92 @@
+package organice.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static organice.logic.commands.MatchCommand.match;
+
+import organice.model.Model;
+import organice.model.person.Donor;
+import organice.model.person.Nric;
+import organice.model.person.Patient;
+import organice.model.person.exceptions.PersonNotFoundException;
+
+public class DoneCommand extends Command {
+    public static final String COMMAND_WORD = "done";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finish the processing of patients and donors"
+            + "Parameters: ic/PATIENT NRIC ic/DONOR NRIC \n"
+            + "Example: " + COMMAND_WORD + " ic/s4512345A ic/s7711123C";
+
+    public static final String MESSAGE_SUCCESS = "Done processing the patient and donor";
+    public static final String MESSAGE_NOT_PROCESSED = "Donor or patient NRIC must be valid";
+
+    private String firstNricString;
+    private String secondNricString;
```

Comment:
Are these variables used in this class?

Reply:
Yes becuase when we done we also have to input donor and patient
Done ic/... ic/....